packages/core/lib/console.js

```diff
@@ -58,18 +58,25 @@ class Console extends EventEmitter {
     });
   }
 
-  start() {
+  async start() {
```

Comment:
Yay, getting rid of callbacks :)

Looks like changing this to `async` won't mess anything up, as it's already `await`ed where used, so that's nice!  However I'd suggest changing `return new Promise(() => {});` to just `return;` now that it's `async`.

Reply:
Ah, but that line is supposed to prevent the `await`-ability! See the hidden first `accept` argument there, which is never called ;)

Reply:
Oh, so the idea is that awaiting it never returns until it gets the exit signal?  Interesting.  Might be worth a comment, I think, because that's pretty non-obvious.  Anyway I'll go hit the approve button, that's up to you. :)